change/@microsoft-fast-foundation-788640ed-cd3b-4d9f-bedc-f179aae99903.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "add activeid to accordion change event",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "chhol@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
I think we need to change this to "prerelease" also.

Reply:
yep, thanks

Reply:
Dangit - I missed pushing this...PR incoming